core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/LimsUtils.java

```diff
@@ -650,25 +651,29 @@ public static boolean isPlainSample(Sample sample) {
 
   public static boolean isIdentitySample(Sample sample) {
     if (!isDetailedSample(sample)) return false;
-    return sample instanceof Identity;
+    return sample instanceof Identity || ((DetailedSample) sample).getSampleClass().getSampleCategory().equals(Identity.CATEGORY_NAME);
```

Comment:
It should return false if there's a null I'd think. Just because it's not yet defined as an Identity doesn't mean it has to be defined at all